packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,403 @@
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: SSWS {apiKey}' \
+```
+
+In the response, locate the ID of the rule and save it somewhere.
+
+### Get the current access policy rule
+
+Use the `policyId` and `ruleId` that you obtained in the previous section to obtain the current access policy rule.
+
+```bash
+  curl --request GET \
+  --url https://${yourOktaDomain}/api/v1/authorizationServers/{AuthorizationServerId}/policies/{policyId}/rules/{ruleId} \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS {apiKey}'
+```
+
+### Add the token exchange grant type to the policy
+
+Next, configure the token exchange grant type for the client that is using the API. You need to update the `grantTypes` parameter by adding the value `urn:ietf:params:oauth:grant-type:token-exchange` so that the token exchange is an allowed grant type in the policy rule.
+
+> **Note:** The `*` value in `scopes` means any scope is valid here.
+
+**Update example**
+
+```json
+curl --location --request PUT
+--url https://${yourOktaDomain}/api/v1/authorizationServers/{AuthorizationServerId}/policies/{policyId}/rules/{ruleId} \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--header 'Authorization: SSWS <apiKey>' \
+-d '{
+	"id": "0pr3erdfwAnLPICrw0g4",
+	"status": "ACTIVE",
+	"name": "allow token exchange",
+	"priority": 1,
+	"created": "2021-04-14T17:47:42.000Z",
+	"lastUpdated": "2021-04-14T17:47:42.000Z",
+	"system": false,
+	"conditions": {
+    	"people": {
+        	"users": {
+            	"include": [],
+            	"exclude": []
+        	},
+        	"groups": {
+            	"include": [
+                	"EVERYONE"
+            	],
+            	"exclude": []
+        	}
+    	},
+    	"grantTypes": {
+        	"include": [
+            	"implicit",
+            	"password",
+				"client_credentials",
+            	"authorization_code",
+            	"urn:ietf:params:oauth:grant-type:token-exchange"
+        	]
+    	},
+    	"scopes": {
+        	"include": [
+            	"*"
+        	]
+    	}
+	},
+	"actions": {
+    	"token": {
+        	"accessTokenLifetimeMinutes": 60,
+        	"refreshTokenLifetimeMinutes": 0,
+        	"refreshTokenWindowMinutes": 10080
+    	}
+	},
+	"type": "RESOURCE_ACCESS"
+}'
+```
+
+## Configure the token exchange grant type for client 2
+
+In this request, use the client ID for client 2 that you created in the "Set up the application" section. Copy the response that is returned from this request for use in the next step.
+
+**Request example**
+
+```bash
+  curl --request GET \
+  --url https://${yourOktaDomain}/oauth2/v1/clients/{clientId}' \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS <apiKey>'
+```
+
+### Update client 2 with the token exchange grant
```

Comment:
This should be done for client 1 as well

Reply:
@venkatviswanathan-okta - was my understanding that for client 1 is optional, not required? @larsjohansen-okta?

If so, then maybe we mention that optionally they can update client 1 with the grant?

Reply:
Fixed. Went with Lar's suggestion.